### PR TITLE
Add a fixture for port toggling

### PR DESCRIPTION
**Pull request by trzhang-msft**
- add a function level fixture to toggle all ports against the randomly selected tor
- fix a few comment errors

### CODE REVIEW

File: tests/common/dualtor/mux_simulator_control.py [hunk 1]
```diff
@@ -311,10 +311,26 @@ def toggle_all_simulator_ports_to_rand_selected_tor(mux_server_url, tbinfo, rand
 
     pytest_assert(_post(mux_server_url, data), "Failed to toggle all ports to {}".format(rand_one_dut_hostname))
 
+@pytest.fixture
+def toggle_all_simulator_ports_against_rand_selected_tor(mux_server_url, tbinfo, rand_one_dut_hostname):
```
Comment (theasianpianist) [inline tests/common/dualtor/mux_simulator_control.py:315]: suggest renaming to `toggle_all_simulator_ports_to_rand_unselected_tor` for consistency with the `rand_unselected_dut` fixture we already have

Comment (trzhang-msft) [reply tests/common/dualtor/mux_simulator_control.py]: updated

**Review by theasianpianist — APPROVED** [review]

**Review by tahmed-dev — APPROVED** [review]

**Review by wangxin — APPROVED** [review]

**Pull request merged**